reference/language-concepts/augments.markdown

```diff
@@ -409,7 +409,7 @@ R: def.my_other_var == Defined ONLY in def.json
 R: def.centos_6_var == Defined ONLY in centos_6.json
 ```
 
-# History
+**History:**
```

Comment:
Nope, it needs to be a heading or it only applies to the last section which is augments. It could be a h2 instead of h1 to put it at the same level as each of the keys.